pkgs/development/compilers/julia/patches/1.6/0001-nix-Remove-library-file-version-numbers-for-JLLs.patch

```diff
@@ -0,0 +1,75 @@
+From a55dfbacedc3cc2026803a70ff05df6507e6c72b Mon Sep 17 00:00:00 2001
+From: Pontus Stenetorp <pontus@stenetorp.se>
+Date: Thu, 8 Apr 2021 05:04:48 +0000
+Subject: [PATCH 1/6] nix: Remove library file version numbers for JLLs
+
+Explicit file names with version numbers prevents shared libraries
+provided by Nix from being picked up at run time.
+---
+ stdlib/GMP_jll/src/GMP_jll.jl         | 4 ++--
+ stdlib/LibGit2_jll/src/LibGit2_jll.jl | 2 +-
+ stdlib/MbedTLS_jll/src/MbedTLS_jll.jl | 6 +++---
+ stdlib/nghttp2_jll/src/nghttp2_jll.jl | 2 +-
+ 4 files changed, 7 insertions(+), 7 deletions(-)
+
+diff --git a/stdlib/GMP_jll/src/GMP_jll.jl b/stdlib/GMP_jll/src/GMP_jll.jl
+index 7a731cb025..b38a78147f 100644
+--- a/stdlib/GMP_jll/src/GMP_jll.jl
++++ b/stdlib/GMP_jll/src/GMP_jll.jl
+@@ -26,8 +26,8 @@ elseif Sys.isapple()
+     const libgmp = "@rpath/libgmp.10.dylib"
+     const libgmpxx = "@rpath/libgmpxx.4.dylib"
+ else
+-    const libgmp = "libgmp.so.10"
+-    const libgmpxx = "libgmpxx.so.4"
++    const libgmp = "libgmp.so"
++    const libgmpxx = "libgmpxx.so"
+ end
+ 
+ function __init__()
+diff --git a/stdlib/LibGit2_jll/src/LibGit2_jll.jl b/stdlib/LibGit2_jll/src/LibGit2_jll.jl
+index 8d1629ed51..0b07766560 100644
+--- a/stdlib/LibGit2_jll/src/LibGit2_jll.jl
++++ b/stdlib/LibGit2_jll/src/LibGit2_jll.jl
+@@ -23,7 +23,7 @@ if Sys.iswindows()
+ elseif Sys.isapple()
+     const libgit2 = "@rpath/libgit2.1.1.dylib"
+ else
+-    const libgit2 = "libgit2.so.1.1"
++    const libgit2 = "libgit2.so"
+ end
+ 
+ function __init__()
+diff --git a/stdlib/MbedTLS_jll/src/MbedTLS_jll.jl b/stdlib/MbedTLS_jll/src/MbedTLS_jll.jl
+index a7ca666b23..b84a42f4e7 100644
+--- a/stdlib/MbedTLS_jll/src/MbedTLS_jll.jl
++++ b/stdlib/MbedTLS_jll/src/MbedTLS_jll.jl
+@@ -31,9 +31,9 @@ elseif Sys.isapple()
+     const libmbedtls = "@rpath/libmbedtls.13.dylib"
+     const libmbedx509 = "@rpath/libmbedx509.1.dylib"
```

Comment:
Wait, did `julia_16` really pass on macOS despite me forgetting to “correct” lines such as these in the patch? Realised this in bed last night and was expecting a build failure when I woke up, but I can not find one on OfBorg.